File: BriefAssistant/Models/ResponseBriefInfo.cs
using System;

namespace BriefAssistant.Models
{
    public class ResponseBriefInfo
    {
        public Guid Id { get; set; }
        public string IssuesPresented { get; set; }
        public string OralArgumentStatement { get; set; }
        public string PublicationStatement { get; set; }
        public string CaseFactsStatement { get; set; }

        public BriefInfo BriefInfo { get; set; }
    }
}

File: BriefAssistant/ClientApp/src/app/briefs/shared/ResponseBriefInfo.ts
import { BriefInfo } from "../../shared/BriefInfo";
import { BriefType } from "../../shared/BriefType";

export class ResponseBriefInfo {
  id: string;
  briefInfo: BriefInfo;
  caseFactsStatement: string;
  oralArgumentStatement: string;
  publicationStatement: string;
  issuesPresented: string;

  constructor() {
    this.briefInfo = new BriefInfo();
    this.caseFactsStatement = "";
    this.oralArgumentStatement = "";
    this.publicationStatement = "";
    this.issuesPresented = "";
    this.briefInfo.type = BriefType.Response;
  }
}
